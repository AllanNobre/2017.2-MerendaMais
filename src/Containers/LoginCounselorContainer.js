import { connect } from 'react-redux';
import { LoginConselheiro } from '../screens';
import { modifyCPF, asyncLoginCounselor } from '../actions/counselorActions';

const mapStatetoProps = state => (
  {
    cpf: state.counselor.cpf,
    message_erro: state.counselor.message_erro,
    isLoading: state.counselor.isLoading,
  }
);

export default connect(mapStatetoProps,
  { modifyCPF, asyncLoginCounselor },
)(LoginConselheiro);
