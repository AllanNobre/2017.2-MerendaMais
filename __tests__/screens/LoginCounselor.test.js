import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// imported as a connected component!
import LoginCounselorContainer from '../../src/Containers/LoginCounselorContainer';


Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {
  application: {
    isLoading: false,
    message_erro: '',
  },
};

const store = mockStore(initialState);

describe('Testing LoginCounselor', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <LoginCounselorContainer />,
      { context: { store } },
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
