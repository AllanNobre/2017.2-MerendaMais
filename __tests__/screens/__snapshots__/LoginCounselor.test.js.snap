// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LoginCounselorContainer renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoginCounselorScreen
    asyncLoginCounselor={[Function]}
    isLoading={false}
    message_erro=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Header
          backButton={false}
          subTitle=""
          title="Merenda +"
        />,
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "flex": 6,
              "justifyContent": "center",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#FAFAFA",
                "borderColor": "gray",
                "borderRadius": 7,
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "paddingLeft": 2,
              }
            }
          >
            <Icon
              allowFontScaling={false}
              color="black"
              name="email"
              size={32}
              style={
                Object {
                  "margin": 5,
                }
              }
            />
            <TextInput
              keyboardType="email-address"
              onChangeText={[Function]}
              placeholder="Email"
              returnKeyType="go"
              underlineColorAndroid="transparent"
              value={undefined}
              width={280}
            />
          </View>
          <TouchableOpacity
            activeOpacity={0.7}
            onPress={[Function]}
            style={
              Object {
                "backgroundColor": "#FF9500",
                "borderRadius": 8,
                "borderWidth": 1,
                "marginBottom": 0,
                "marginTop": 50,
                "paddingHorizontal": 133,
                "paddingVertical": 18,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "white",
                  "fontSize": 20,
                }
              }
            >
              Entrar
            </Text>
          </TouchableOpacity>
          <TouchableOpacity
            activeOpacity={0.6}
            onPress={[Function]}
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Object {
                  "marginTop": 30,
                }
              }
            >
              É um presidente?
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "blue",
                  }
                }
              >
                 Clique aqui
              </Text>
            </Text>
          </TouchableOpacity>
        </View>,
        <View
          style={
            Object {
              "alignItems": "center",
              "borderTopColor": "#a9a9a9",
              "borderTopWidth": 1,
              "flex": 0.7,
              "justifyContent": "center",
            }
          }
        >
          <TouchableOpacity
            activeOpacity={0.6}
            onPress={[Function]}
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Ainda não se cadastrou?
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "blue",
                  }
                }
              >
                 Cadastrar-se
              </Text>
            </Text>
          </TouchableOpacity>
        </View>,
      ],
      "style": Object {
        "flex": 1,
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "backButton": false,
          "subTitle": "",
          "title": "Merenda +",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FAFAFA",
                  "borderColor": "gray",
                  "borderRadius": 7,
                  "borderWidth": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "paddingLeft": 2,
                }
              }
            >
              <Icon
                allowFontScaling={false}
                color="black"
                name="email"
                size={32}
                style={
                  Object {
                    "margin": 5,
                  }
                }
              />
              <TextInput
                keyboardType="email-address"
                onChangeText={[Function]}
                placeholder="Email"
                returnKeyType="go"
                underlineColorAndroid="transparent"
                value={undefined}
                width={280}
              />
            </View>,
            <TouchableOpacity
              activeOpacity={0.7}
              onPress={[Function]}
              style={
                Object {
                  "backgroundColor": "#FF9500",
                  "borderRadius": 8,
                  "borderWidth": 1,
                  "marginBottom": 0,
                  "marginTop": 50,
                  "paddingHorizontal": 133,
                  "paddingVertical": 18,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  }
                }
              >
                Entrar
              </Text>
            </TouchableOpacity>,
            <TouchableOpacity
              activeOpacity={0.6}
              onPress={[Function]}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "marginTop": 30,
                  }
                }
              >
                É um presidente?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Clique aqui
                </Text>
              </Text>
            </TouchableOpacity>,
          ],
          "style": Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "flex": 6,
            "justifyContent": "center",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <Icon
                  allowFontScaling={false}
                  color="black"
                  name="email"
                  size={32}
                  style={
                    Object {
                      "margin": 5,
                    }
                  }
                />,
                <TextInput
                  keyboardType="email-address"
                  onChangeText={[Function]}
                  placeholder="Email"
                  returnKeyType="go"
                  underlineColorAndroid="transparent"
                  value={undefined}
                  width={280}
                />,
              ],
              "style": Object {
                "alignItems": "center",
                "backgroundColor": "#FAFAFA",
                "borderColor": "gray",
                "borderRadius": 7,
                "borderWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "paddingLeft": 2,
              },
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "allowFontScaling": false,
                  "color": "black",
                  "name": "email",
                  "size": 32,
                  "style": Object {
                    "margin": 5,
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "keyboardType": "email-address",
                  "onChangeText": [Function],
                  "placeholder": "Email",
                  "returnKeyType": "go",
                  "underlineColorAndroid": "transparent",
                  "value": undefined,
                  "width": 280,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "activeOpacity": 0.7,
              "children": <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  }
                }
              >
                Entrar
              </Text>,
              "onPress": [Function],
              "style": Object {
                "backgroundColor": "#FF9500",
                "borderRadius": 8,
                "borderWidth": 1,
                "marginBottom": 0,
                "marginTop": 50,
                "paddingHorizontal": 133,
                "paddingVertical": 18,
              },
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": "Entrar",
                "disabled": false,
                "ellipsizeMode": "tail",
                "style": Object {
                  "color": "white",
                  "fontSize": 20,
                },
              },
              "ref": null,
              "rendered": "Entrar",
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "activeOpacity": 0.6,
              "children": <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "marginTop": 30,
                  }
                }
              >
                É um presidente?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Clique aqui
                </Text>
              </Text>,
              "onPress": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": Array [
                  "É um presidente?",
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "blue",
                      }
                    }
                  >
                     Clique aqui
                  </Text>,
                ],
                "disabled": false,
                "ellipsizeMode": "tail",
                "style": Object {
                  "marginTop": 30,
                },
              },
              "ref": null,
              "rendered": Array [
                "É um presidente?",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "accessible": true,
                    "allowFontScaling": true,
                    "children": " Clique aqui",
                    "disabled": false,
                    "ellipsizeMode": "tail",
                    "style": Object {
                      "color": "blue",
                    },
                  },
                  "ref": null,
                  "rendered": " Clique aqui",
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <TouchableOpacity
            activeOpacity={0.6}
            onPress={[Function]}
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Ainda não se cadastrou?
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "blue",
                  }
                }
              >
                 Cadastrar-se
              </Text>
            </Text>
          </TouchableOpacity>,
          "style": Object {
            "alignItems": "center",
            "borderTopColor": "#a9a9a9",
            "borderTopWidth": 1,
            "flex": 0.7,
            "justifyContent": "center",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeOpacity": 0.6,
            "children": <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Ainda não se cadastrou?
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "blue",
                  }
                }
              >
                 Cadastrar-se
              </Text>
            </Text>,
            "onPress": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "accessible": true,
              "allowFontScaling": true,
              "children": Array [
                "Ainda não se cadastrou?",
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Cadastrar-se
                </Text>,
              ],
              "disabled": false,
              "ellipsizeMode": "tail",
            },
            "ref": null,
            "rendered": Array [
              "Ainda não se cadastrou?",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "accessible": true,
                  "allowFontScaling": true,
                  "children": " Cadastrar-se",
                  "disabled": false,
                  "ellipsizeMode": "tail",
                  "style": Object {
                    "color": "blue",
                  },
                },
                "ref": null,
                "rendered": " Cadastrar-se",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Header
            backButton={false}
            subTitle=""
            title="Merenda +"
          />,
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "flex": 6,
                "justifyContent": "center",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#FAFAFA",
                  "borderColor": "gray",
                  "borderRadius": 7,
                  "borderWidth": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "paddingLeft": 2,
                }
              }
            >
              <Icon
                allowFontScaling={false}
                color="black"
                name="email"
                size={32}
                style={
                  Object {
                    "margin": 5,
                  }
                }
              />
              <TextInput
                keyboardType="email-address"
                onChangeText={[Function]}
                placeholder="Email"
                returnKeyType="go"
                underlineColorAndroid="transparent"
                value={undefined}
                width={280}
              />
            </View>
            <TouchableOpacity
              activeOpacity={0.7}
              onPress={[Function]}
              style={
                Object {
                  "backgroundColor": "#FF9500",
                  "borderRadius": 8,
                  "borderWidth": 1,
                  "marginBottom": 0,
                  "marginTop": 50,
                  "paddingHorizontal": 133,
                  "paddingVertical": 18,
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "white",
                    "fontSize": 20,
                  }
                }
              >
                Entrar
              </Text>
            </TouchableOpacity>
            <TouchableOpacity
              activeOpacity={0.6}
              onPress={[Function]}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "marginTop": 30,
                  }
                }
              >
                É um presidente?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Clique aqui
                </Text>
              </Text>
            </TouchableOpacity>
          </View>,
          <View
            style={
              Object {
                "alignItems": "center",
                "borderTopColor": "#a9a9a9",
                "borderTopWidth": 1,
                "flex": 0.7,
                "justifyContent": "center",
              }
            }
          >
            <TouchableOpacity
              activeOpacity={0.6}
              onPress={[Function]}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Ainda não se cadastrou?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Cadastrar-se
                </Text>
              </Text>
            </TouchableOpacity>
          </View>,
        ],
        "style": Object {
          "flex": 1,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "backButton": false,
            "subTitle": "",
            "title": "Merenda +",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#FAFAFA",
                    "borderColor": "gray",
                    "borderRadius": 7,
                    "borderWidth": 1,
                    "flexDirection": "row",
                    "justifyContent": "center",
                    "paddingLeft": 2,
                  }
                }
              >
                <Icon
                  allowFontScaling={false}
                  color="black"
                  name="email"
                  size={32}
                  style={
                    Object {
                      "margin": 5,
                    }
                  }
                />
                <TextInput
                  keyboardType="email-address"
                  onChangeText={[Function]}
                  placeholder="Email"
                  returnKeyType="go"
                  underlineColorAndroid="transparent"
                  value={undefined}
                  width={280}
                />
              </View>,
              <TouchableOpacity
                activeOpacity={0.7}
                onPress={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FF9500",
                    "borderRadius": 8,
                    "borderWidth": 1,
                    "marginBottom": 0,
                    "marginTop": 50,
                    "paddingHorizontal": 133,
                    "paddingVertical": 18,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "white",
                      "fontSize": 20,
                    }
                  }
                >
                  Entrar
                </Text>
              </TouchableOpacity>,
              <TouchableOpacity
                activeOpacity={0.6}
                onPress={[Function]}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "marginTop": 30,
                    }
                  }
                >
                  É um presidente?
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "blue",
                      }
                    }
                  >
                     Clique aqui
                  </Text>
                </Text>
              </TouchableOpacity>,
            ],
            "style": Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "flex": 6,
              "justifyContent": "center",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <Icon
                    allowFontScaling={false}
                    color="black"
                    name="email"
                    size={32}
                    style={
                      Object {
                        "margin": 5,
                      }
                    }
                  />,
                  <TextInput
                    keyboardType="email-address"
                    onChangeText={[Function]}
                    placeholder="Email"
                    returnKeyType="go"
                    underlineColorAndroid="transparent"
                    value={undefined}
                    width={280}
                  />,
                ],
                "style": Object {
                  "alignItems": "center",
                  "backgroundColor": "#FAFAFA",
                  "borderColor": "gray",
                  "borderRadius": 7,
                  "borderWidth": 1,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "paddingLeft": 2,
                },
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "allowFontScaling": false,
                    "color": "black",
                    "name": "email",
                    "size": 32,
                    "style": Object {
                      "margin": 5,
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "keyboardType": "email-address",
                    "onChangeText": [Function],
                    "placeholder": "Email",
                    "returnKeyType": "go",
                    "underlineColorAndroid": "transparent",
                    "value": undefined,
                    "width": 280,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeOpacity": 0.7,
                "children": <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "white",
                      "fontSize": 20,
                    }
                  }
                >
                  Entrar
                </Text>,
                "onPress": [Function],
                "style": Object {
                  "backgroundColor": "#FF9500",
                  "borderRadius": 8,
                  "borderWidth": 1,
                  "marginBottom": 0,
                  "marginTop": 50,
                  "paddingHorizontal": 133,
                  "paddingVertical": 18,
                },
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "accessible": true,
                  "allowFontScaling": true,
                  "children": "Entrar",
                  "disabled": false,
                  "ellipsizeMode": "tail",
                  "style": Object {
                    "color": "white",
                    "fontSize": 20,
                  },
                },
                "ref": null,
                "rendered": "Entrar",
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeOpacity": 0.6,
                "children": <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "marginTop": 30,
                    }
                  }
                >
                  É um presidente?
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "blue",
                      }
                    }
                  >
                     Clique aqui
                  </Text>
                </Text>,
                "onPress": [Function],
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "accessible": true,
                  "allowFontScaling": true,
                  "children": Array [
                    "É um presidente?",
                    <Text
                      accessible={true}
                      allowFontScaling={true}
                      disabled={false}
                      ellipsizeMode="tail"
                      style={
                        Object {
                          "color": "blue",
                        }
                      }
                    >
                       Clique aqui
                    </Text>,
                  ],
                  "disabled": false,
                  "ellipsizeMode": "tail",
                  "style": Object {
                    "marginTop": 30,
                  },
                },
                "ref": null,
                "rendered": Array [
                  "É um presidente?",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "accessible": true,
                      "allowFontScaling": true,
                      "children": " Clique aqui",
                      "disabled": false,
                      "ellipsizeMode": "tail",
                      "style": Object {
                        "color": "blue",
                      },
                    },
                    "ref": null,
                    "rendered": " Clique aqui",
                    "type": [Function],
                  },
                ],
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <TouchableOpacity
              activeOpacity={0.6}
              onPress={[Function]}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Ainda não se cadastrou?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Cadastrar-se
                </Text>
              </Text>
            </TouchableOpacity>,
            "style": Object {
              "alignItems": "center",
              "borderTopColor": "#a9a9a9",
              "borderTopWidth": 1,
              "flex": 0.7,
              "justifyContent": "center",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "activeOpacity": 0.6,
              "children": <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Ainda não se cadastrou?
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  disabled={false}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "blue",
                    }
                  }
                >
                   Cadastrar-se
                </Text>
              </Text>,
              "onPress": [Function],
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": Array [
                  "Ainda não se cadastrou?",
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    disabled={false}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "blue",
                      }
                    }
                  >
                     Cadastrar-se
                  </Text>,
                ],
                "disabled": false,
                "ellipsizeMode": "tail",
              },
              "ref": null,
              "rendered": Array [
                "Ainda não se cadastrou?",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "accessible": true,
                    "allowFontScaling": true,
                    "children": " Cadastrar-se",
                    "disabled": false,
                    "ellipsizeMode": "tail",
                    "style": Object {
                      "color": "blue",
                    },
                  },
                  "ref": null,
                  "rendered": " Cadastrar-se",
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
    },
  },
}
`;
